### DISCUSSION TITLE
Can't login to space canvas

### DISCUSSION THREAD

**Discussion by MacaylaMarvelous81** [deta/cloud-docs#702] [Help]
Hi! I tried to login to alpha.space for the first time, but it always redirects me to the login page again. I still encounter this issue on other devices, private/incognito browser windows, and when I clear my cookies. Here I can login to space fine, but not alpha.space:

https://user-images.githubusercontent.com/29156241/210273648-6c2afb24-a64a-4794-8710-bdb18fb77775.mp4

(p.s. Sorry, I didn't realize email wasn't for support)

**Comment by MacaylaMarvelous81** [comment]
Update: It seems like now, in the same browser and device in the video, Space ends up reloading/redirecting to the Canvas over and over. In a private window and on other devices, the old issue of redirecting to the login page still occurs.

**Reply by sudo-subg** [reply]
Try removing the cookies for all `deta.app` domains.

**Reply by MacaylaMarvelous81** [reply]
It doesn't look like I have any cookies under deta.app

**Comment by abdelhai** [comment]
go to alpha.deta.space (don't use deta.space)
in general, we're working on merging both in the next few days, so expect some light issue. feel free to report them on our discord

**Comment by MacaylaMarvelous81** [comment]
Since the new space version was merged, I can't use deta.space anymore for the same reason. old.deta.space does work, but I hear that it'll be going away in a few weeks.

**Reply by MacaylaMarvelous81** [reply]
It seems to only occur on the root (https://deta.space/), but not anywhere else. For example (https://deta.space/foo) opens the page without redirecting to itself. Discovery works too, but I can't install anything.

**Comment by MacaylaMarvelous81** [comment]
I think that it was an account issue. I created a new account and everything's working fine. It's about time I made a new account anyways.
